code/modules/admin/admin_verbs.dm

```diff
@@ -91,6 +91,7 @@ GLOBAL_LIST_INIT(admin_verbs_fun, list(
 	/client/proc/cmd_admin_dress,
 	/client/proc/cmd_admin_gib_self,
 	/client/proc/drop_bomb,
+	/client/proc/give_disease,
```

Comment:
Is this new, or left over from something else?

Reply:
its been in the code but the proc itself wasn't being used.  I forgot to remove it again but we could just leave it?

Reply:
As long as it works and isn't just a broken proc, that's fine.